### PR TITLE
LED remap tables stored as bytearrays, saves a good chunk of RAM

### PR DESCRIPTION
**Pull request by PaintYourDragon**
Original tables are kept as comments in source if needed for future reference

### CODE REVIEW

**Review by ladyada — APPROVED** [review]
did you check if its faster?

**Comment by PaintYourDragon** [comment]
I haven’t benchmarked, but subjectively it did seem better. Can definitely confirm that RAM use is at least 2K smaller though.

While working in the Arduino lib, noticed that R/G/B order was wrong (now fixed there) and have confirmed similar issues here. These are NOT fixed yet. Can either merge this now for the byte fixes, or hold off ’til I fix the color order as well.

File: adafruit_is31fl3741/adafruit_ledglasses.py [hunk 1]
```diff
@@ -22,38 +22,65 @@
...
+    #    (260, 261, 266),  # 16
+    #    (348, 349, 262),  # 17
+    #    (233, 234, 235),  # 18
+    #    (237, 238, 239),  # 19
+    #    (339, 340, 232),  # 20
+    #    (327, 328, 329),  # 21
+    #    (305, 91, 90),  # 22
+    #    (296, 61, 60),  # 23
+    # )
+    ledmap_bytes = bytearray(
```
Comment (dhalbert) [inline adafruit_is31fl3741/adafruit_ledglasses.py:59]: Style note: these don't need to be `bytearray`s; they can just be `bytes`, and you don't even need to say `bytes`:
```py
ledmap_bytes = (
        b"\x01\x1F\x00\x1F\x00\x1E"
        b"\x01\x16\x00\x01\x00\x00"
...
)

**Review by dhalbert — CHANGES_REQUESTED** [review]
Glad this helped. I noted a further simplification.

**Comment by ladyada** [comment]
@PaintYourDragon up to u!

File: adafruit_is31fl3741/adafruit_ledglasses.py (hunk 1 above)
Comment (dhalbert) [reply adafruit_is31fl3741/adafruit_ledglasses.py:59]: Just take away the `bytearray` word, that's easy.

Comment (PaintYourDragon) [reply adafruit_is31fl3741/adafruit_ledglasses.py:59]: Ah! Thank you. I’ll make that change along with the RGB order stuff then, will commit again later.

**Pull request merged**